microsoft-365/admin/activity-reports/onedrive-for-business-usage.md

```diff
@@ -52,7 +52,7 @@ You can get a view into OneDrive for Business usage by looking at the **Accounts
 |2.  <br/> |The data in each report usually covers up to the last 24 to 48 hours. <br/>|
 |3.  <br/> |The **Accounts** view shows the trend in the number of total and active OneDrive accounts. "Active accounts" are any in which users view, modify, upload, download, share, or sync files.  <br/> |
 |4.  <br/> |The **Files** view shows the number of number of total and active files. A file is considered active if it has been saved, synced, modified or shared within the specific time period.  <br/> NOTE: A file activity can occur multiple times for a single file, but will count only as one active file. For example, you can save and sync the same file multiple times over a specified time period, but it will count only as one single active file and one single synced file in the data.           |
```

Comment:
```suggestion
|4.  <br/> |The **Files** view shows the number of total and active files. A file is considered active if it has been saved, synced, modified, or shared within a specific time period.  <br/> NOTE: A file activity can occur multiple times for a single file, but will count only as one active file. For example, you can save and sync the same file multiple times over a specified time period, but it will count only as one single active file and one single synced file in the data.           |
```